File: scripts/commands/generic/cmdGrantSkill.js
function executeCommand(galacticManager, player, target, args) {
	var creature = player.getCreatureObject();
	var GrantSkillIntent = Java.type("intents.experience.GrantSkillIntent");
	var IntentType = Java.type("intents.experience.GrantSkillIntent.IntentType");
	
	new GrantSkillIntent(IntentType.GRANT, args, creature, true).broadcast();
}

File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/generic/CmdGrantSkill.java
package com.projectswg.holocore.resources.support.global.commands.callbacks.generic;

import com.projectswg.holocore.intents.gameplay.player.experience.skills.GrantSkillIntent;
import com.projectswg.holocore.resources.support.global.commands.ICmdCallback;
import com.projectswg.holocore.resources.support.global.player.Player;
import com.projectswg.holocore.resources.support.objects.swg.SWGObject;
import org.jetbrains.annotations.NotNull;

public final class CmdGrantSkill implements ICmdCallback {
	
	@Override
	public void execute(@NotNull Player player, SWGObject target, @NotNull String args) {
		GrantSkillIntent.broadcast(GrantSkillIntent.IntentType.GRANT, args, player.getCreatureObject(), true);
	}
	
}
